.github/workflows/swift-toolchain.yml

```diff
@@ -361,7 +361,6 @@ jobs:
         run: Copy-Item ${{ github.workspace }}\SourceCache\swift-build\cmake\SQLite\CMakeLists.txt -destination ${{ github.workspace }}\SourceCache\sqlite-3.43.2\CMakeLists.txt
 
       - uses: compnerd/gha-setup-vsdevenv@main
-        if: matrix.os == 'Windows'
```

Comment:
Why not send this upstream?

Reply:
👍🏾 I'll upstream nearly all of this and resend for review once the PR checks are passing again.

Reply:
Upstreamed as https://github.com/compnerd/swift-build/pull/754/files and will rebase when merged